src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -3067,6 +3067,121 @@ public void ValidateGraphQLSchemaForCircularReference(string schema)
             Assert.AreEqual(DataApiBuilderException.SubStatusCodes.ErrorInInitialization, exception.SubStatusCode);
         }
 
+        /// <summary>
+        /// GraphQL Schema types defined -> Character and Planet
+        /// DAB runtime config entities defined -> Planet(Not defined: Character)
+        /// Mismatch of entities and types between provided GraphQL schema file and DAB config results in actionable error message.
+        /// </summary>
+        /// <exception cref="ApplicationException"></exception>
```

Comment:
comment not accurate.. what is Star for? it doesn't look used in this test.

Reply:
`Star` is just another entity and it had nothing to with this test, hence I didn't mention it in the doc. Anyway I removed it from the test to avoid the confusion. Another test is added to cover alias scenario using `Star` entity.